packages/api-mcp/src/cli.ts

```diff
@@ -0,0 +1,49 @@
+import minimist from "minimist";
+
+import { createConsoleLogger } from "@recallnet/api-sdk/mcp-server/console-logger.js";
+
+import { config, validateEnv } from "./env.js";
+import { startServer } from "./server.js";
+
+/**
+ * Parse command line arguments and run the server
+ */
+export async function run(): Promise<void> {
+  // Validate environment
+  const missingVars = validateEnv();
+  if (missingVars.length > 0) {
+    console.error(
+      `Missing required environment variables: ${missingVars.join(", ")}`,
+    );
+    process.exit(1);
+  }
+
+  const argv = minimist(process.argv.slice(2), {
```

Comment:
should we keep the flag definitions the same as the env vars we define later? e.g., `--server-url` & `SERVER_URL`, etc